backend/services/installer/installTask.go

```diff
@@ -37,9 +37,9 @@ func (instance *Task) IsInstalling() bool {
 
 // function that sets the current task status
 func (instance *Task) setStatus(status global.AppStatus) {
-	if instance.Status == status {
```

Comment:
@bonhokage06 why this was commented? thanks

Reply:
hi sir @jhoe123 It needs to disable becuase when we click cancel it will not update the status of the instance. thank you